processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -225,7 +239,7 @@ private static ColumnCapabilities getEffectiveCapabilities(
   public static Long convertObjectToLong(Object valObj)
```

Comment:
This method doesn't match `convertObjectToDouble()` in null handling of the result of getExactLongFromDecimalString().